rasa/server.py

```diff
@@ -468,7 +468,7 @@ def create_app(
         if not events:
             logger.warning(
```

Comment:
mhm, they should be able to do sth about it since they are probably the ones who are doing the API request?

Reply:
makes sense